openwisp-monitoring/files/monitoring.agent

```diff
@@ -182,6 +184,13 @@ send_data() {
 					# remove saved data
 					rm -f "$filename"
 					break
+				elif [ "$response_code" = "400" ];then
+					logger -s "Data not sent successfully. Response code is \"$response_code\"." \
+					    "Data may be tempered, discarding data" \
+						-t openwisp-monitoring \
+						-p daemon.err
+					rm -f "$filename"
```

Comment:
Error message is logged in non-verbose mode too since it will be discarded and data points will be missing on controller end